<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>638</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>萍方-简</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>端口扫描器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="8" column="0" colspan="4">
       <widget class="QTextEdit" name="tEdtResult">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="4">
       <widget class="QProgressBar" name="pBarProgress">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="lEdtHost">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="maxLength">
         <number>15</number>
        </property>
        <property name="placeholderText">
         <string>127.0.0.1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>IP地址</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>扫描结果</string>
        </property>
       </widget>
      </item>
      <item row="9" column="3">
       <widget class="QPushButton" name="pBtnExit">
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="lEdtEndPort">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>5</number>
        </property>
        <property name="placeholderText">
         <string>550</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>结束端口</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>开始端口</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lEdtStartPort">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>5</number>
        </property>
        <property name="placeholderText">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QPushButton" name="pBtnPause">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>暂停扫描</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>超时(ms)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="pBtnStartScan">
        <property name="text">
         <string>开始扫描</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lEdtTimeout">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>6</number>
        </property>
        <property name="placeholderText">
         <string>50</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>线程数</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="lEdtThreadNum">
        <property name="maxLength">
         <number>4</number>
        </property>
        <property name="placeholderText">
         <string>256</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lEdtHost</tabstop>
  <tabstop>lEdtStartPort</tabstop>
  <tabstop>lEdtEndPort</tabstop>
  <tabstop>lEdtTimeout</tabstop>
  <tabstop>lEdtThreadNum</tabstop>
  <tabstop>pBtnStartScan</tabstop>
  <tabstop>pBtnPause</tabstop>
  <tabstop>tEdtResult</tabstop>
  <tabstop>pBtnExit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
